rules/cairo.json

```diff
@@ -39,6 +39,17 @@
           "distribution": "sle"
         }
       ]
+    },
+    {
+      "packages": [
+        "mingw-w64-x86_64-cairo",
+        "mingw-w64-i686-cairo"
+      ],
+      "constraints": [
+        {
+          "os": "windows"
```

Comment:
Should we add a contraint for `> R 4.0` to avoid special case logic in RSPM?

Reply:
I've been thinking about that too, but don't have a solid opinion yet. We could also go back to using generic package names like `cairo` and push it to the clients, which would either run `pacman -S mingw-w64-{i686,x86_64}-cairo` on R 4.0+, or find the rwinlib libs from https://github.com/rwinlib/cairo on R 3.x. 

Since we don't even have any R 3.x rules yet, I'd say we just wait and see. Whatever we go with, it'd be nice to match how sysreqsdb intends to do Windows in the future as well: https://github.com/r-hub/sysreqsdb#supported-platforms